src/consumers/mod.rs

```diff
@@ -87,7 +87,7 @@ impl<T> ScopedMessage<T> {
     /// This method doesn't have an error because if you nack, it means the message needs to be
     /// rehandled, so the ack wait will eventually expire
     pub async fn nack(&mut self) {
-        if let Err(e) = self.custom_ack(AckKind::Nak).await {
+        if let Err(e) = self.custom_ack(AckKind::Nak(None)).await {
```

Comment:
What's this? I don't see anything contained in the enum value upstream

Reply:
This is from the latest 0.29 nats. It now takes a duration to wait before sending NAK